site/troubleshooting.md

```diff
@@ -20,7 +20,7 @@ Check the weave container logs with
 
 A reasonable amount of information, and all errors, get logged there.
 
-The log verbosity can be increased by supplying the `-debug` flag when
+The log verbosity can be increased by supplying the flag `-log-level=debug` when
```

Comment:
to clarify: you are suggesting to document it as `--log-level`, even though the code and the usage message will show `-log-level` ?  It seems to be impossible to have single-minus and double-minus options in the same program, without much bigger changes.